readme.md

```diff
@@ -159,6 +159,10 @@ Deletes a single document matching `filter`. Resolves with the number of documen
 
 Deletes multiple documents matching `filter`. Resolves with the number of documents deleted.
 
+#### `aggregate(pipeline[, opts]): Promise<object[]>`
+
+Transforms the document according to the multi-stage `pipeline`. Resolves with the aggregated objects.
```

Comment:
I think that "the document" is ambiguous here 🤔 what is it really that will be transformed?

Maybe something like this: (copied from the MongoDB documentation 😁)

```suggestion
Executes an aggregation framework pipeline against the collection. Resolves with the aggregated objects.
```